RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -403,6 +400,10 @@ float npcStoreDiscountMulti()
 	{
 		retval -= 0.05;
 	}
+	else if(possessEquipment($item[designer sweatpants]) && auto_is_valid($item[designer sweatpants]))
+	{
+		retval -= 0.05;
+	}
 	
```

Comment:
Adding designer sweatpants case since they can be used in G-Lover while travoltan trousers cannot be.